config/rbac/vsphereclustertemplate_editor_role.yaml

```diff
@@ -0,0 +1,24 @@
+# permissions for end users to edit vsphereclustertemplates.
```

Comment:
This file is not required, could you please remove this.

Reply:
The file `config/rbac/role.yaml` gets updated once we make changes to the controllers to allow them to read/update/delete/reconcile the template objects.